staging/src/k8s.io/client-go/tools/cache/reflector.go

```diff
@@ -174,8 +195,9 @@ var internalPackages = []string{"client-go/tools/cache/"}
 func (r *Reflector) Run(stopCh <-chan struct{}) {
 	klog.V(2).Infof("Starting reflector %s (%s) from %s", r.expectedTypeName, r.resyncPeriod, r.name)
 	wait.BackoffUntil(func() {
-		if err := r.ListAndWatch(stopCh); err != nil {
-			utilruntime.HandleError(err)
+		err := r.ListAndWatch(stopCh)
+		if err != nil {
```

Comment:
Why did you change the scoping of err?

Reply:
haha, this PR has gone thru a bunch of revisions, and this change was from an earlier one where it made more sense. changed back!